packages/bun/README.md

```diff
@@ -0,0 +1,62 @@
+<p align="center">
+  <a href="https://sentry.io/?utm_source=github&utm_medium=logo" target="_blank">
+    <img src="https://sentry-brand.storage.googleapis.com/sentry-wordmark-dark-280x84.png" alt="Sentry" width="280" height="84">
+  </a>
+</p>
+
+# Official Sentry SDK for Bun
```

Comment:
m: Should we add a note around this SDK bein experimental? Not sure what our strategy is here but I'd give it a few weeks of alpha/beta time until we promote it to stable, WDYT?